repos/fdbt-site/src/pages/api/defineCapStart.ts

```diff
@@ -0,0 +1,56 @@
+import { NextApiResponse } from 'next';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { CAP_START_ATTRIBUTE } from '../../constants/attributes';
+import { redirectToError, redirectTo } from '../../utils/apiUtils';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { CapStartInfo, DayOfTheWeek } from '../../../src/interfaces/matchingJsonTypes';
+import { isCapStart, isDayOfTheWeek } from '../../../src/interfaces/typeGuards';
+
+export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
+    try {
+        const errors: ErrorInfo[] = [];
+        let startValue: DayOfTheWeek | undefined;
+        const { capStart, start } = req.body;
+        if (isCapStart(capStart)) {
+            if (capStart === 'fixedWeekdays') {
+                if (!isDayOfTheWeek(start)) {
+                    errors.push({
+                        id: 'start',
+                        errorMessage: 'Start value is invalid',
```

Comment:
```suggestion
                        errorMessage: 'Select a start day',
```

We also need to use the new function I mentioned above here.